docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
+
+### Tutorial
+
```

Comment:
It's worth adding a screenshot of what ToolbarView looks like, because its frequently used but the name is obscure.